src/lib/db/role-store.ts

```diff
@@ -141,6 +148,15 @@ export default class RoleStore implements IRoleStore {
             .orWhere('type', 'project');
     }
 
+    async getProjectRolesCount(): Promise<number> {
+        return this.db
+            .from<IRole>(T.ROLES)
+            .count('*')
+            .where('type', 'custom')
+            .orWhere('type', 'project')
```

Comment:
Do we want to include the project roles here? If I'm not going crazy, you can't add or remove them so it might just be noise here

Reply:
Actually I already do this, so this code is not used at all. I will remove it for simplicity.